codegen/golang.go

```diff
@@ -12,6 +12,7 @@ import (
 
 	"github.com/cloudquery/plugin-sdk/schema"
 	"github.com/iancoleman/strcase"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
we shouldn't import anything plugin specific in the SDK. working on a solution.

Reply:
This is the related PR https://github.com/cloudquery/plugin-sdk/pull/139